streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/system/service/impl/MenuServiceImpl.java

```diff
@@ -133,7 +137,9 @@ public void updateMenu(Menu menu) throws Exception {
     public void deleteMenus(String[] menuIds) throws Exception {
         for (String menuId : menuIds) {
             // Find users associated with these menus/buttons
-            this.baseMapper.deleteRoleMenuByMenuId(Long.parseLong(menuId));
+            LambdaQueryWrapper<RoleMenu> queryWrapper = new LambdaQueryWrapper<>();
+            queryWrapper.eq(StringUtils.isNotEmpty(menuId), RoleMenu::getMenuId, Long.parseLong(menuId));
```

Comment:
Code could be more concise: ` queryWrapper.eq(RoleMenu::getMenuId, menuId);`

Reply:
Done